spec/features/get_integrations_spec.rb

```diff
@@ -1,17 +1,47 @@
-describe "Get integrations dot file" do
+describe "Get integrations" do
   before do
-    td.create_pact_with_hierarchy("Foo", "1", "Bar")
-      .create_verification(provider_version: "2")
+    td.create_consumer("Foo")
+      .create_provider("Bar")
+      .create_integration
+      .create_consumer("Apple")
+      .create_provider("Pear")
+      .create_integration
+      .create_consumer("Dog")
+      .create_provider("Cat")
+      .create_integration
   end
 
   let(:path) { "/integrations" }
-  let(:response_body_hash) { JSON.parse(subject.body, symbolize_names: true) }
-
-  subject { get path, nil, {"HTTP_ACCEPT" => "application/hal+json" }  }
+  let(:query) { nil }
+  let(:response_body_hash) { JSON.parse(subject.body) }
+  subject { get path, query, {"HTTP_ACCEPT" => "application/hal+json" }  }
 
   it { is_expected.to be_a_hal_json_success_response }
 
   it "returns a json body with embedded integrations" do
-    expect(JSON.parse(subject.body)["_embedded"]["integrations"]).to be_a(Array)
+    expect(response_body_hash["_embedded"]["integrations"]).to be_a(Array)
+  end
+
+  context "with pagination options" do
+    let(:query) { { "pageSize" => "2", "pageNumber" => "1" } }
+
+    it_behaves_like "a paginated response"
```

Comment:
@Inksprout I've pulled this out into https://github.com/pact-foundation/pact_broker/blob/master/spec/support/shared_examples_for_responses.rb#L12